src/NZGBplugin/Ui_NewFeatureDialog.py

```diff
@@ -1,21 +1,8 @@
-#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 
-################################################################################
-#
-#  New Zealand Geographic Board gazetteer application,
-#  Crown copyright (c) 2020, Land Information New Zealand on behalf of
-#  the New Zealand Government.
-#
-#  This file is released under the MIT licence. See the LICENCE file found
-#  in the top-level directory of this distribution for more information.
-#
-################################################################################
-
-
 # Form implementation generated from reading ui file 'Ui_NewFeatureDialog.ui'
 #
-# Created by: PyQt5 UI code generator 5.9.2
+# Created by: PyQt5 UI code generator 5.10.1
```

Comment:
Just wondering.. does a generated file belong to the repository at all ? (something for later, probably)